src/main/java/edu/cornell/kfs/sys/identity/ExcludeSubmitterSeparationOfDutiesRoleTypeService.java

```diff
@@ -0,0 +1,49 @@
+package edu.cornell.kfs.sys.identity;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.codec.binary.StringUtils;
```

Comment:
I would suggest using the Commons Lang3 version of StringUtils instead, for consistency with the rest of our code.

Reply:
good point, not sure how that got imported. Updating